subsys/usb/device_next/class/usbd_cdc_ncm.c

```diff
@@ -33,7 +33,7 @@ LOG_MODULE_REGISTER(cdc_ncm, CONFIG_USBD_CDC_NCM_LOG_LEVEL);
 
 enum {
 	CDC_NCM_IFACE_UP,
-	CDC_NCM_CLASS_ENABLED,
+	CDC_NCM_DATA_IFACE_ENABLED,
```

Comment:
In "usb: device_next: do not stop when OUT transfer has zero length" commit message you cannot really refer to the commit by its SHA, because the SHA will change when the patches enter mainline. The solution would be to split that away from this PR and update the commit message after the underlying commit is in main.

Reply:
Yes, I screwed up again. Changed commit message.